web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -56,6 +57,29 @@ const sizeText = (size) => {
   return filesize(size, { base: 2 });
 };
 
+const AutoCalculatedTooltip = ({ volume, children }) => {
+  // the size is not affected by snapshots or other volumes
+  if (!volume.snapshotsAffectSizes && volume.sizeRelevantVolumes.length === 0) {
+    const content = <Text>These limits are not affected by any other settings</Text>;
+    return <Tooltip content={content}>{children}</Tooltip>;
+  }
+
+  const content = (
+    <>
+      <Text>These limits are affected by:</Text><br />
+      <List>
+        {volume.snapshotsAffectSizes &&
+          <ListItem>The configuration of snapshots</ListItem>}
+        {volume.sizeRelevantVolumes.length > 0 &&
+          <ListItem>By presence of other volumes ({volume.sizeRelevantVolumes.join(", ")})</ListItem>}
+      </List>
+    </>
```

Comment:
NP: Please, do not use a hard-coded line break `<br />`. Instead, wrap these elementes with a `div` using the `stack` CSS class.

```suggestion
    <div className="stack">
      <Text>These limits are affected by:</Text>
      <List>
        {volume.snapshotsAffectSizes &&
          <ListItem>The configuration of snapshots</ListItem>}
        {volume.sizeRelevantVolumes.length > 0 &&
          <ListItem>By presence of other volumes ({volume.sizeRelevantVolumes.join(", ")})</ListItem>}
      </List>
    </div>
```

Reply:
Also, I'd use a simple `<p>` instead of `<Text>` because I feel the latter does not provide too much value at this time and I tend to use plain HTML when possible.